src/components/MapView/Analyser/index.tsx

```diff
@@ -0,0 +1,402 @@
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  Input,
+  LinearProgress,
+  Radio,
+  RadioGroup,
+  Switch,
+  TextField,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+
+import { faCaretDown, faChartBar } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { useDispatch, useSelector } from 'react-redux';
+import bbox from '@turf/bbox';
+
+import DatePicker from 'react-datepicker';
+
+import {
+  getBoundaryLayerSingleton,
+  LayerDefinitions,
+} from '../../../config/utils';
+import {
+  AggregationOperations,
+  BoundaryLayerProps,
+  NSOLayerProps,
+  WMSLayerProps,
+  LayerKey,
+} from '../../../config/types';
+import { LayerData } from '../../../context/layers/layer-data';
+import { layerDataSelector } from '../../../context/mapStateSlice/selectors';
+import { Extent } from '../Layers/raster-utils';
+import { availableDatesSelector } from '../../../context/serverStateSlice';
+import {
+  AnalysisDispatchParams,
+  analysisResultSelector,
+  clearAnalysisResult,
+  isAnalysisLayerActiveSelector,
+  isAnalysisLoadingSelector,
+  requestAndStoreAnalysis,
+  setIsMapLayerActive,
+} from '../../../context/analysisResultStateSlice';
+import AnalysisTable from './AnalysisTable';
+import LayerDropdown from '../../../utils/LayerDropdown';
+
+const boundaryLayer = getBoundaryLayerSingleton();
+
+function Analyser({ classes }: AnalyserProps) {
+  const dispatch = useDispatch();
+  const boundaryLayerData = useSelector(layerDataSelector(boundaryLayer.id)) as
+    | LayerData<BoundaryLayerProps>
+    | undefined;
+
+  const availableDates = useSelector(availableDatesSelector);
+  const analysisResult = useSelector(analysisResultSelector);
+
+  const isAnalysisLoading = useSelector(isAnalysisLoadingSelector);
+  const isMapLayerActive = useSelector(isAnalysisLayerActiveSelector);
+
+  const [isAnalyserFormOpen, setIsAnalyserFormOpen] = useState(false);
+  const [isTableViewOpen, setIsTableViewOpen] = useState(true);
+
+  // form elements
+  const [hazardLayerId, setHazardLayerId] = useState<LayerKey>();
+  const [statistic, setStatistic] = useState(AggregationOperations.Mean);
+  const [baselineLayerId, setBaselineLayerId] = useState<LayerKey>();
+  const [selectedDate, setSelectedDate] = useState<number | null>(null);
+
+  const [belowThreshold, setBelowThreshold] = useState('');
+  const [aboveThreshold, setAboveThreshold] = useState('');
+  const [thresholdError, setThresholdError] = useState<string | null>(null);
+
+  // enforce errors
+  useEffect(() => {
+    const belowThresholdValue = parseFloat(belowThreshold);
+    const aboveThresholdValue = parseFloat(aboveThreshold);
+    if (belowThresholdValue < aboveThresholdValue) {
+      setThresholdError('Min threshold is larger than Max!');
+    } else {
+      setThresholdError(null);
+    }
+  }, [belowThreshold, aboveThreshold]);
```

Comment:
got it. Since the threshold is currently the only field with aggressive validation, I'll set up a specialised onChange handler with validation for threshold fields.